### PR TITLE
ui: translations

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: apps/ios/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -936,7 +936,7 @@
 "Clear verification" = "Сбросить подтверждение";
 
 /* No comment provided by engineer. */
-"Color chats with the new themes." = "Добавьте цвета к чатам в настройках.";
+"Color chats with the new themes." = "Добавьте цвета к чатам в настройках тем.";
```
Comment (epoberezkin) [inline apps/ios/ru.lproj/Localizable.strings:939]:
```suggestion
"Color chats with the new themes." = "Добавьте цвета к чатам в настройках.";
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 2]
```diff
@@ -1459,7 +1459,7 @@
 "Delete queue" = "Удаление очереди";
 
 /* No comment provided by engineer. */
-"Delete up to 20 messages at once." = "Удаляйте до 20 сообщений за раз.";
+"Delete up to 20 messages at once." = "Удаляйте до 20 сообщений одновременно.";
```
Comment (epoberezkin) [inline apps/ios/ru.lproj/Localizable.strings:1462]:
```suggestion
"Delete up to 20 messages at once." = "Удаляйте до 20 сообщений за раз.";
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 3]
```diff
@@ -3358,7 +3358,7 @@
 "Private message routing" = "Конфиденциальная доставка сообщений";
 
 /* No comment provided by engineer. */
-"Private message routing 🚀" = "Конфиденциальная доставка 🚀";
+"Private message routing 🚀" = "Конфиденциальная доставка сообщений 🚀";
```
Comment (epoberezkin) [inline apps/ios/ru.lproj/Localizable.strings:3361]:
```suggestion
"Private message routing 🚀" = "Конфиденциальная доставка 🚀";
```

File: apps/multiplatform/common/src/commonMain/resources/MR/hu/strings.xml [hunk 4]
```diff
@@ -2022,7 +2022,7 @@
     <string name="cant_call_member_send_message_alert_text">Üzenet küldése a hívások engedélyezéséhez.</string>
     <string name="you_need_to_allow_calls">Engedélyeznie kell a hívásokat az ismerőse számára, hogy fel tudják hívni egymást.</string>
     <string name="you_can_still_view_conversation_with_contact">A(z) %1$s nevű ismerősével folytatott beszélgetéseit továbbra is megtekintheti a csevegések listájában.</string>
-    <string name="compose_message_placeholder">Üzenet</string>
+    <string name="compose_message_placeholder">Üzenet...</string>
```
Comment (epoberezkin) [inline apps/multiplatform/common/src/commonMain/resources/MR/hu/strings.xml:2025]:
```suggestion
    <string name="compose_message_placeholder">Üzenet</string>
```

**Comment by kumo63** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**